File: server-SpringBoot2/enuocms-spring-business/src/main/java/com/enuocms/business/model/Category.java
package com.enuocms.business.model;


import com.enuocms.core.model.BaseEntity;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.ToString;

import javax.persistence.Table;

/**
 * Created by zhanxiaoping on 2017/8/28.
 * zhanxp@me.com
 * category
 */
@Data
@EqualsAndHashCode(callSuper = true)
@ToString
@Table(name = "category")
public class Category extends BaseEntity {
    private String title;
}

File: server-dotNet/Com.EnuoCms.Model/Entities/Category.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

/**
 * Created by zhanxiaoping 
 * zhanxp@me.com
 */
namespace Com.EnuoCms.Model.Entities
{
  [Table("category")]
  public class Category : BaseEntity
  {
    //[Key]
    //[Column("id")]
    //public int ID { get; set; }

    [Required]
    [MaxLength(200)]
    [Column("title")]
    [Display(Name = "名称")]
    public string Title { get; set; }
  }
}
